traffic_prophet/countmatch/reader.py

```diff
@@ -291,6 +305,47 @@ def get_zipreader(self, zipname):
                            'data': data,
                            'year': data.at[0, 'Timestamp'].year}
 
+    def read_sql(self):
+        """Read PostgreSQL table contents into AnnualCount objects."""
+        # ptcs and sttcs hold arrays of processed counts.
+        ptcs = {}
+        sttcs = {}
+
+        # Cycle through all zip files.
+        for year in range(cfg.cm['min_year'], cfg.cm['max_year'] + 1):
+            current_counts = [AnnualCount.from_raw_data(c)
+                              for c in self.get_sqlreader(year)]
+
+            # Append counts to processed count dicts.
+            self.append_counts(current_counts, ptcs, sttcs)
+
+        self.check_processed_count_integrity(ptcs, sttcs)
+        self.unify_counts(ptcs, sttcs)
+        self.ptcs = ptcs
+        self.sttcs = sttcs
+
+    def get_sqlreader(self, year):
+        with self.source.connect() as db_con:
+            all_data = pd.read_sql(
+                self._sql_cmd.format(dbt=self.source.tablename,  year=year),
```

Comment:
This is legit the only place you use `_sql_cmd` so might as well just attach it here.  Not like Postgres uses newlines as syntax anyway.